pkg/controllers/statusaggregator/plugins/deployment.go

```diff
@@ -74,11 +74,8 @@ func (receiver *DeploymentPlugin) AggregateStatues(
 
 		deployStatus := &appsv1.DeploymentStatus{}
 		if err = util.ConvertViaJson(status, deployStatus); err != nil {
-			klog.Errorf(
-				"Failed to convert the status of cluster resource object deployment %q for cluster %q",
-				sourceObject.GetName(),
-				clusterName,
-			)
+			logger.WithValues("cluster-name", clusterName).
```

Comment:
nit: we can create a sublogger with `cluster-name` key at the start of the for loop.